classes/Kohana/Model/Auth/User.php

```diff
@@ -73,7 +66,7 @@ public static function initialize(Jam_Meta $meta)
 
 	public static function unique_key($value)
 	{
-		return Valid::email($value) ? 'email' : ((is_numeric($value) OR $value === NULL) ? 'id' : 'username');
+		return (is_numeric($value) OR $value === NULL) ? 'id' : 'email';
```

Comment:
@ivank Do you think we can keep that logic if the behaviour is enabled? This way we would at least not break anything if the behaviour is added to a model.